2021-11-17 23:40:41,463 INFO    MainThread:24264 [wandb_setup.py:_flush():71] setting env: {}
2021-11-17 23:40:41,464 INFO    MainThread:24264 [wandb_setup.py:_flush():71] setting login settings: {'anonymous': 'must'}
2021-11-17 23:40:41,464 INFO    MainThread:24264 [wandb_init.py:_log_setup():357] Logging user logs to /home/nicolas/workspace_josue/Pytorch-UNet/wandb/run-20211117_234041-3bdenyx6/logs/debug.log
2021-11-17 23:40:41,464 INFO    MainThread:24264 [wandb_init.py:_log_setup():358] Logging internal logs to /home/nicolas/workspace_josue/Pytorch-UNet/wandb/run-20211117_234041-3bdenyx6/logs/debug-internal.log
2021-11-17 23:40:41,465 INFO    MainThread:24264 [wandb_init.py:init():390] calling init triggers
2021-11-17 23:40:41,465 INFO    MainThread:24264 [wandb_init.py:init():397] wandb.init called with sweep_config: {}
config: {}
2021-11-17 23:40:41,465 INFO    MainThread:24264 [wandb_init.py:init():435] starting backend
2021-11-17 23:40:41,466 INFO    MainThread:24264 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-17 23:40:41,511 INFO    MainThread:24264 [backend.py:ensure_launched():199] starting backend process...
2021-11-17 23:40:41,533 INFO    MainThread:24264 [backend.py:ensure_launched():205] started backend process with pid: 24284
2021-11-17 23:40:41,534 INFO    MainThread:24264 [wandb_init.py:init():444] backend started and connected
2021-11-17 23:40:41,535 INFO    MainThread:24264 [wandb_init.py:init():503] updated telemetry
2021-11-17 23:40:41,587 INFO    MainThread:24264 [wandb_init.py:init():533] communicating current version
2021-11-17 23:40:41,874 INFO    MainThread:24264 [wandb_init.py:init():538] got version response 
2021-11-17 23:40:41,874 INFO    MainThread:24264 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-11-17 23:40:42,248 INFO    MainThread:24264 [wandb_init.py:init():576] starting run threads in backend
2021-11-17 23:40:43,190 INFO    MainThread:24264 [wandb_run.py:_console_start():1693] atexit reg
2021-11-17 23:40:43,191 INFO    MainThread:24264 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.REDIRECT
2021-11-17 23:40:43,191 INFO    MainThread:24264 [wandb_run.py:_redirect():1572] Redirecting console.
2021-11-17 23:40:43,193 INFO    MainThread:24264 [wandb_run.py:_redirect():1628] Redirects installed.
2021-11-17 23:40:43,193 INFO    MainThread:24264 [wandb_init.py:init():603] run started, returning control to user process
2021-11-17 23:40:43,193 INFO    MainThread:24264 [wandb_run.py:_config_callback():911] config_cb None None {'epochs': 5, 'batch_size': 1, 'learning_rate': 1e-05, 'val_percent': 0.1, 'save_checkpoint': True, 'img_scale': 0.5, 'amp': True}
2021-11-17 23:40:43,323 INFO    MainThread:24264 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 1
2021-11-17 23:40:43,324 INFO    MainThread:24264 [wandb_run.py:_restore():1635] restore
2021-11-17 23:40:45,744 INFO    MainThread:24264 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 773
  total_bytes: 773
}

2021-11-17 23:40:45,967 INFO    MainThread:24264 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 773
  total_bytes: 773
}

2021-11-17 23:40:46,263 INFO    MainThread:24264 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 773
  total_bytes: 6060
}

2021-11-17 23:40:46,367 INFO    MainThread:24264 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 773
  total_bytes: 6787
}

2021-11-17 23:40:46,470 INFO    MainThread:24264 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6787
  total_bytes: 6787
}

2021-11-17 23:40:46,574 INFO    MainThread:24264 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6787
  total_bytes: 6787
}

2021-11-17 23:40:46,677 INFO    MainThread:24264 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6787
  total_bytes: 6787
}

2021-11-17 23:40:46,779 INFO    MainThread:24264 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6787
  total_bytes: 6787
}

2021-11-17 23:40:46,882 INFO    MainThread:24264 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6787
  total_bytes: 6787
}

2021-11-17 23:40:46,986 INFO    MainThread:24264 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6787
  total_bytes: 6787
}

2021-11-17 23:40:47,193 INFO    MainThread:24264 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6787
  total_bytes: 6787
}

2021-11-17 23:40:47,583 INFO    MainThread:24264 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6787
  total_bytes: 6787
}
local_info {
}

2021-11-17 23:40:48,641 INFO    MainThread:24264 [wandb_run.py:_append_files():2061] logging synced files
